### ISSUE TITLE
NVDA tracebacks and uses a lot of memory when using the browser

### ISSUE THREAD

**Issue by just-anotheruser** [nvaccess/nvda#13763]
### Steps to reproduce:
Open a browser page, Crome, brave, mozilla. All work.
Navigate around the page, try to use the arrows, home and end, read the text letter by letter both in edit fields or out.
### Actual behavior:
NVDA's log gives the following traceback:
ERROR - eventHandler.executeEvent (13:52:21.955) - MainThread (4780):
error executing event: caret on <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible object at 0x04CAAF10> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "browseMode.pyc", line 1345, in event_caret
  File "eventHandler.pyc", line 107, in next
  File "NVDAObjects\behaviors.pyc", line 249, in event_caret
  File "NVDAObjects\IAccessible\__init__.pyc", line 1621, in event_caret
  File "NVDAObjects\__init__.pyc", line 1205, in event_caret
  File "review.pyc", line 174, in handleCaretMove
  File "documentBase.pyc", line 22, in makeTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 120, in __init__
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 232, in _findContentDescendant
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 32, in __call__
_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))
ERROR - scriptHandler.executeScript (14:05:24.758) - MainThread (5940):
error executing script: <bound method EditableText.script_caret_moveByLine of <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible object at 0x013C3DD0>> with gesture 'up arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 216, in executeScript
  File "editableText.pyc", line 224, in script_caret_moveByLine
  File "editableText.pyc", line 160, in _caretMovementScriptHelper
  File "editableText.pyc", line 93, in _hasCaretMoved
  File "documentBase.pyc", line 22, in makeTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 120, in __init__
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 232, in _findContentDescendant
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 32, in __call__
_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))

When more of them fill the log my pc starts being overloaded. from its usual 4.7 physical gb usage it jumps nearly to its limit, to around 13. same with the virtual memory. Restarting nvda drops the memory usage to its normal patterns.
### Expected behavior:
Preferably no tracebacks and less memory usage.
#### NVDA installed/portable/running from source:
NVDA is installed.
#### NVDA version:
2022.1
#### Windows version:
Windows 10 21H2 (10.0.19044
#### Name and version of other software in use when reproducing the issue:
NVDA and a browser page, tested with crome, brave mozilla latest versions.
#### Other information about your system:
Lenovo Laptop
Processor	AMD Ryzen 7 5800H with Radeon Graphics            3.20 GHz
Installed RAM	16.0 GB (13.9 GB usable)
System type	64-bit operating system, x64-based processor
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, the same thing was happening with the previous NVDA version.
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by CyrilleB79** [comment]
By chance, any link with the Waves MaxxAudio driver described in [In-Process 18th May 2022](https://www.nvaccess.org/post/in-process-18th-may-2022/)?

I know it is not the same brand of computer, but who knows?

**Comment by Qchristensen** [comment]
Not sure if related, but I've had a user on Twitter report extremely high memory use from NVDA (advised 3.9gb on a system with 4gb).

Machine is a Lenovo Ideapad 320 - 15 AST AMD model with dolby audio powered by Real Technology hd audio (no log or other info supplied yet)

**Comment by seanbudd** [comment]
Welcome @just-anotheruser ,

We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need an example website. Ideally provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.
- Could you also provide the version of Chrome and Firefox you are using.

Kind Regards,
NV Access Software Developers

**Comment by CyrilleB79** [comment]
@just-anotheruser you wrote:
> @CyrilleB79 No, I use Realtech audio drivers, the ones that it was purchased with. Didn't install other audio drivers.

Actually the issue seems to come from the Realtek audio driver. See on [this page](https://www.dell.com/support/kbdoc/en-uk/000183963/high-cpu-usage-from-wavemaxx-when-using-internal-mic-speakers?lwp=rt); even if it is for DELL computers, it is indicated that the cause is the Realtek audio driver.
Please check that you have a more recent driver than the one indicated in the article or that you have disabled [the Waves MaxxAudio service](https://www.nvaccess.org/post/in-process-18th-may-2022/).
Let us know.

@storyofablindlife could you perform the same checks?

**Comment by storyofablindlife** [comment]
@CyrilleB79 I contacted Lenovo and they said that the only version they support for now for this driver is what i currently have,
not what is apparently needed which is 
version 6.0.9107.1.
I currently have version
6.0.9088.1.
I asked them if they can look into things on their side but. Well, I doubt they will.
I pointed them here too just incase.
I will try to see if i can get another driver or something to work when I will have someone sighted around if my audio breaks.
The waves Maxx can't be an issue on my laptop at least because I don't have it so that's not the problem here.
